mobile/src/main/java/org/openhab/habdroid/util/Util.java

```diff
@@ -185,6 +187,43 @@ public static void setActivityTheme(@NonNull final Activity activity, String the
         return R.style.HABDroid_Light;
     }
 
+    /**
+     * If fullscreen is enabled and we are on at least android 4.4 set
+     * the system visibility to fullscreen + immersive + noNav
+     *
+     * @author Dan Cunningham
+     */
+    public static void checkFullscreen(Activity activity) {
+        checkFullscreen(activity, isFullscreenEnabled(activity));
+    }
+
+    public static void checkFullscreen(Activity activity, boolean isEnabled) {
+        int uiOptions = activity.getWindow().getDecorView().getSystemUiVisibility();
+        if (isEnabled) {
+            uiOptions = uiOptions
+                    | View.SYSTEM_UI_FLAG_HIDE_NAVIGATION
+                    | View.SYSTEM_UI_FLAG_IMMERSIVE_STICKY
+                    | View.SYSTEM_UI_FLAG_FULLSCREEN;
+        } else {
+            uiOptions = uiOptions
+                    & View.SYSTEM_UI_FLAG_HIDE_NAVIGATION
+                    & View.SYSTEM_UI_FLAG_IMMERSIVE_STICKY
+                    & View.SYSTEM_UI_FLAG_FULLSCREEN;
```

Comment:
This seems off. It should rather be `uiOptions &= ~(...flags...)`

Suggestion:
```
final int flags = View.SYSTEM_UI_FLAG_HIDE_NAVIGATION
        | View.SYSTEM_UI_FLAG_IMMERSIVE_STICKY
        | View.SYSTEM_UI_FLAG_FULLSCREEN;
if (isEnabled) {
    uiOptions |= flags;
} else {
    uiOptions &= ~flags;
}
```